app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/AttachmentActionsBottomSheetDialog.kt

```diff
@@ -64,29 +67,38 @@ class AttachmentActionsBottomSheetDialog : ActionsBottomSheetDialog() {
             return@with
         }
 
+        if (attachment is SwissTransferFile) {
+            with(binding) {
+                openWithItem.isVisible = false
+                kDriveItem.isVisible = false
```

Comment:
We always favorise the "positive" option

```suggestion
                openWithItem.isGone = true
                kDriveItem.isGone = true
```